packages/apps/reputation-oracle/server/src/modules/escrow-completion/results-processing/escrow-results-processor.spec.ts

```diff
@@ -86,6 +86,37 @@ describe('BaseEscrowResultsProcessor', () => {
       ).rejects.toThrow(testError);
     });
 
+    it('should throw if escrow is not found', async () => {
+      const chainId = generateTestnetChainId();
+      const escrowAddress = faker.finance.ethereumAddress();
+
+      const baseResultsUrl = faker.internet.url();
+      mockedGetIntermediateResultsUrl.mockResolvedValueOnce(baseResultsUrl);
+
+      const resultsUrl = `${baseResultsUrl}/${faker.system.fileName()}`;
+      processor.constructIntermediateResultsUrl.mockReturnValueOnce(resultsUrl);
+
+      const resultsFileContent = Buffer.from(faker.lorem.sentence());
+      mockedStorageService.downloadFile.mockResolvedValueOnce(
+        resultsFileContent,
+      );
+
+      processor.assertResultsComplete.mockResolvedValueOnce(undefined);
+
+      mockedEscrowUtils.getEscrow.mockResolvedValueOnce(
+        null as unknown as IEscrow,
```

Comment:
```suggestion
        null,
```
It should be possible to resolve just null if types are fixed